tests/test_ax_integration.py

```diff
@@ -113,6 +117,114 @@ def test_generation_method(self):
         self.assertEqual(n_sobol, correct_n_sobol)
         self.assertEqual(n_opt, correct_n_opt)
 
+class AxOrdinalGPTestCase(unittest.TestCase):
```

Comment:
I wonder if instead of copy-pasting this test case for every model if we could make subclasses or pass in different arguments using [parameterized](https://pypi.org/project/parameterized/). But I am worried about making this test take too long to run, so I think by default we should only run the test using the continuous model (the current integration test).

Reply:
I'd still like to see these integration tests generalized, but if that is not possible, you should at least put a `@unittest.skip` decorator like in the other PR.